src/Shouldly/MessageGenerators/ShouldBeginEndWithMessageGenerator.cs

```diff
@@ -16,28 +16,12 @@ public override string GenerateErrorMessage(IShouldlyAssertionContext context)
             var codePart = context.CodePart;
             var expected = context.Expected.ToStringAwesomely();
             var actualValue = context.Actual.ToStringAwesomely();
-            string actual;
-            if (codePart == actualValue)
-            {
-                if (context.IsNegatedAssertion)
-                {
-                    actual = " did";
-                }
-                else
-                {
-                    actual = " did not";
-                }
-            }
-            else
-            {
-                actual = $@" was
-{actualValue}";
-            }
-
+            var actual = codePart == actualValue
+                ? context.IsNegatedAssertion ? " did" : " did not" : $@" was {actualValue}";
```

Comment:
This removed the intentional newline that is part of the message, and I imagine this is the reason there are failing tests.